test/middleware.test.js

```diff
@@ -0,0 +1,1455 @@
+import fs from 'fs';
+import path from 'path';
+
+import express from 'express';
+import request from 'supertest';
+import MemoryFileSystem from 'memory-fs';
+
+import middleware from '../src';
+
+import getCompiler from './helpers/getCompiler';
+
+import { mockRequest, mockResponse } from './mock-express';
+
+import webpackConfig from './fixtures/server-test/webpack.config';
+import webpackMultiConfig from './fixtures/server-test/webpack.array.config';
+import webpackQuerystringConfig from './fixtures/server-test/webpack.querystring.config';
+import webpackClientServerConfig from './fixtures/server-test/webpack.client.server.config';
+import webpackErrorConfig from './fixtures/error-config/webpack.config';
+import webpackWarningConfig from './fixtures/warning-config/webpack.config';
+
+describe('middleware', () => {
+  let instance;
+  let listen;
+  let app;
+
+  const logLevel = 'error';
+
+  function listenShorthand(done) {
+    return app.listen(8000, '127.0.0.1', (err) => {
+      if (err) {
+        return done(err);
+      }
+
+      return done();
```

Comment:
`return` is unnecessary

Reply:
It is for `consistent-return` rule, also returning same type in function is good practice